scripts/build/index.js

```diff
@@ -244,8 +244,8 @@ const buildServiceImages = async () => {
 
   console.log('working directory', workingDir);
   for (const service of versions.SERVICES) {
-    await exec('cp', ['-r', `${workingDir}/node_modules`, service]);
-    await exec('cp', ['-r', './shared-libs', service]);
+    await exec('rsync', ['-r', '-p', './shared-libs', service]);
+    await exec('rsync', ['-r', '-p', `${workingDir}/node_modules`, service]);
```

Comment:
@m5r This is how I found to keep the link reference, the `rsync` is available in the containers. 
What do you think? Is it okay?

Reply:
`rsync` does the job well but it's not always available. If this script is only ever executed in docker images, I think this is fine. But if this gets executed on the host machine, it would be safer to make it work with `cp`

I can't give it a shot locally at the moment but have you tried some of the `cp` options mentioned [here](https://superuser.com/a/138588)? I think one of them could solve our symlink issue here

Reply:
yes, I tried `cp -Pr`, `cp -r -a`, but it doesn't work in mac at least